Sorts/AlphaNumericalSort.js

```diff
@@ -0,0 +1,10 @@
+/*
+  alpha numerical sort = Natural sort order 
```

Comment:
Add more in-depth comments, explaining the algorithm.

Reply:
yeah, you a right) thx

====
updated